dotCMS/src/main/java/com/dotcms/rest/api/v1/contenttype/ContentTypeHelper.java

```diff
@@ -35,6 +35,7 @@
 import java.util.List;
 import java.util.Locale;
 import java.util.Map;
```

Comment:
There are a couple of private fields in this class that could be removed. They are not used 
 private final WebResource webResource;
 private final ContentTypeUtil contentTypeUtil;

Reply:
Done